alkira/resource_alkira_credential_pan_instance.go

```diff
@@ -16,6 +16,7 @@ func resourceAlkiraCredentialPanInstance() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			State: schema.ImportStatePassthrough,
 		},
+		DeprecationMessage: "alkira_credential_pan_instance has been deprecated. Please specify name, auth_code and auth_key directly in instance block of resource service_pan. See documentation for example.",
```

Comment:
Do we still care about the credential name? I think it will be generated automatically?

Reply:
API requires an instance name to be present. I got a validation error. "name must not be blank"